src/actions/videoAction.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const getVideos = () => async (videoDispatch) => {
+    try {
+        const { data } = await axios.get('/api/videos');
+        videoDispatch({type: 'SUCCESS', payload: data.videos});
```

Comment:
One suggestion use can use payload = {videos: data.videos} ie you can make payload an object as it is a good practice if you want to give multiple values with one single payload.

Reply:
Here, I am just getting videos only - I Guess there is no need to do it. Maybe use in future cases and thanks for suggesting this way.